website/src/content/docs/docs/emitters/clients/advanced-configurations.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Advanced Client Emitter Configurations
+---
+
+import { Aside } from "@astrojs/starlight/components";
+
+## Advanced Configurations for Client Emitters
```

Comment:
is it really advanced configuration, do you have another part about basic configuration? Feels like that should just be common configuration

Reply:
Rename the doc to be common configuration